docs/docs/ci.md

```diff
@@ -0,0 +1,138 @@
+These images come with the `gha-tools` installed.
+
+## ci-imgs
+
+The RAPIDS CI Docker images are defined in the [ci-imgs repository](https://github.com/rapidsai/ci-imgs/).
+The base Dockerfiles for the images install core dependencies required for all RAPIDS builds.
+
+### wheel images
+
+For wheels, the Dockerfile installs the necessary version of Python using [`pyenv`](https://github.com/pyenv/pyenv), while all the main build dependencies (compilers, CMake, system libraries, etc) are installed using the OS's package manager.
+RAPIDS attempts to build wheels that are as portable as possible (in compliance with the [manylinux standard](https://peps.python.org/pep-0513/)), which means that the binaries must be compiled on the oldest supported toolchain and bundle all external library dependencies so as to not throw inscrutable errors when installed onto a user system that is missing those libraries or has ABI-incompatible versions of those libraries installed.
+Note that we do not use the upstream [PyPA manylinux images](https://github.com/pypa/manylinux/).
+Those do not contain CUDA, and attempting to maintain a fork of those images with CUDA proved quite challenging.
+Instead, we select an OS that we know has the glibc version associated with the manylinux standard that we wish to comply with, then choose [NVIDIA Docker images](https://hub.docker.com/r/nvidia/cuda) for that OS as the basis of our CI images.
+An appropriate OS by manylinux version may be found using [this handy website](https://github.com/mayeut/pep600_compliance).
+
+For wheels, separate images are created for build and testing.
+While the build images contain all libraries required for building RAPIDS, the test image is slimmed down and is missing some of these libraries.
+This allows us to test (at least to some extent) our manylinux compliance by verifying that our packages run even without those extra libraries installed.
+In an ideal world the testing image would not even have the CTK installed, but that is not currently possible because RAPIDS has some dependencies (numba and cupy) that are not as self-contained and still rely on a system CUDA installation.
+Until that changes, we cannot remove the CTK from wheel testing images.
+
+### conda images
+
+For conda, the images are different depending on whether they are for CUDA 11 or for CUDA 12+.
+As described in [the conda section above](packaging.md#conda) the available packages differed between these versions.
+Therefore, in RAPIDS conda CI images, for CUDA 11 images the CTK is installed on the system, while for CUDA 12 images it is not.
+The installation of conda itself in our CI images is handled upstream in the miniforge-cuda images (see below).
+
+The ci-imgs repository is automatically rebuild whenever new changes are pushed to any of the following:
+
+- dependency-file-generator
+- miniforge-cuda
+- gha-tools
```

Comment:
```suggestion
The [ci-imgs](https://github.com/rapidsai/ci-imgs) repository is automatically rebuild whenever new changes are pushed to any of the following:

- [dependency-file-generator](https://github.com/rapidsai/dependency-file-generator)
- [miniforge-cuda](https://github.com/rapidsai/miniforge-cuda)
- [gha-tools](https://github.com/rapidsai/gha-tools)
```

Could you please make all of these links?

For example, I think this is the first time in this documentation that `gha-tools` has been linked. Might not be immediately clear to someone new that this is a repo in the `rapidsai` GitHub org.

Reply:
Ah yeah so this is a mixed bag:
- ci-imgs is linked above. This is a subsection of the ci-imgs section, so no need to relink.
- dependency-file-generator is covered on a separate page that is technically sequenced before, but there is obviously no guarantee that someone will read this in sequence. Not linking here is an artifact of when I originally had one big doc before I split it up. I'll link to the dfg section in the docs instead.
- miniforge-cuda is just below this. I'll just add a "see below"
- gha-tools is in the previous section. If you think that's too far away and it might be missed, we could link back to the section, but I feel like it's good enough that it's in the same page since someone just doing Ctrl + F on the page will find it. WDYT?

Reply:
Yep all works for me! I think it's right to not assume a specific sequence of reading these docs.

The linking and parentheticals you added in https://github.com/rapidsai/build-planning/pull/55/commits/f92209a043c7117005c054f75edcd22eb94558ce address what I was going for with this comment, I'm good with this. Thank you!